src/test/java/com/researchspace/webapp/integrations/fieldmark/FieldmarkMVCIT.java

```diff
@@ -0,0 +1,219 @@
+package com.researchspace.webapp.integrations.fieldmark;
+
+import com.google.common.io.Files;
+import com.researchspace.api.v1.controller.API_MVC_TestBase;
+import com.researchspace.webapp.integrations.fieldmark.model.FieldmarkNotebook;
+import com.researchspace.webapp.integrations.fieldmark.model.FieldmarkRecordsJsonExport;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.web.client.RestTemplate;
+
+@WebAppConfiguration
+public class FieldmarkMVCIT extends API_MVC_TestBase {
+
+  private static final String COMMA_DELIMITER = ",";
+
+  @Before
+  public void setUp() throws Exception {
+    super.setUp();
+  }
+
+  @Test
+  @Ignore(
+      "This test was used for the Fieldmark POC. "
+          + "We leave the test Ignored so we can potentially run it manually by the bearer token")
+  public void testCreateDatasetAndPushFile() throws IOException, URISyntaxException {
+    RestTemplate restTemplate = new RestTemplate();
+    HttpHeaders headers = new HttpHeaders();
+    headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
+
+    /* ************
+     ///// SET THE ACCESS TOKEN MANUALLY -  It is in BitWarden /////
+    **************** */
+    String ACCESS_TOKEN = "-----------PASTE_TOKEN_HERE_FROM_BITWARDEN----------";
```

Comment:
Bitwarden is our company-specific tool, maybe more generic instructions about obtaining token form Fieldmark would be better for external users who may read it.